cw/cw305/capture.py

```diff
@@ -361,6 +361,34 @@ def aes_fvsr_key(ctx: typer.Context,
     capture_end(ctx.obj.cfg)
 
 
+@app_capture.command()
+def aes_mix_column(ctx: typer.Context,
+                   num_traces: int = opt_num_traces,
+                   plot_traces: int = opt_plot_traces):
+    """Capture AES traces. Fixed key, Random texts. 4 sets of traces. Mix Column HD CPA Attack.
+    Attack implemented by ChipWhisperer:
+    Repo: https://github.com/newaetech/chipwhisperer-jupyter/blob/master/experiments/...
+    ...MixColumn%20Attack.ipynb
```

Comment:
Here you don't need to wrap the line as it breaks the URL. You can add the following tag at the very end of the line to not make the lint tool complain: `# noqa: E501`